app/data/stacks/reth/README.md

```diff
@@ -0,0 +1,65 @@
+# Reth
+Deploy a Reth API node alongside Lighthouse.
+
+## Clone required repositories
+
+```
+$ laconic-so --stack reth setup-repositories
+```
+
+## Build the fixturenet-eth containers
```

Comment:
I assume this is a typo and it should be "Build the reth containers"

Reply:
Correct, thanks... fixed